mandatory/2-function-creation.js

```diff
@@ -6,7 +6,15 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (!isNaN(num) && num % 2 === 0 && num <= 100) {
+    return true;
+  }  else {
+    return false;
+  }
+}
+
+console.log(validate("name"));
 
```

Comment:
This is quite complicated! It's really common to get in a muddle like this at this point of the course, so don't despair!

There are a few small things here that might be useful to clear up:

1. An expression that evaluates to true is effectively the same as true. You can return the result of the expression directly and you don't need to also return true. Think of the expression as a door. If the expression evaluates to true then the door opens and the code contained within the block is executed. 

2. Try taking out the else block and just putting `return false` instead. What happens? Why? Why not?